pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile

```diff
@@ -35,13 +36,13 @@ Map cypressFlowParameters = [
   tenantId                : 'diku',
   adminUsername           : 'diku_admin',
   adminPassword           : 'admin',
-  parallelExecParameters  : '--env grepTags="smoke criticalPath extendedPath --nonParallel"',
-  sequentialExecParameters: '--env grepTags="smoke+nonParallel criticalPath+nonParallel extendedPath+nonParallel"',
-  testsTimeout            : '10',
+  parallelExecParameters  : '--env grepTags="smoke criticalPath --nonParallel"',
+  sequentialExecParameters: '--env grepTags="smoke+nonParallel criticalPath+nonParallel"',
+  testsTimeout            : '14',
   testrailProjectID       : '14',
   testrailRunID           : '2108',
   numberOfWorkers         : '10',
-  agent                   : 'rancher||jenkins-agent-java11'
+  agent                   : 'jenkins-agent-java11'
```

Comment:
Are we sure that this workers will be 10?

Reply:
That value hasn't been changed for a long time, so I guess, we are sure.